docs/recording_provenance.md

```diff
@@ -194,44 +194,44 @@ the result of an operation should await the
  Chronicle will notify commit completion in 2 stages, the first will be a submit
  notification like:
 
- ```json
+```json
 {
   "stage": "SUBMIT",
   "error": null,
   "delta": null,
   "txId": "12d3236ae1b227391725d2d9315b7ca53747217c5d..",
 }
- ```
+```
 
  And the second, when the Chronicle transaction has been committed to the
  ledger:
 
- ```json
+```json
 {
-  "stage": "COMMIT",
```

Comment:
I have my doubts about the re-spacing of this JSON.

Reply:
The reason for the editing was that the formatting completely off in the compiled docs. This PR at least fixes that issue, so now we have an actual code block. How would you change things now?

Reply:
Looking at it, I see the JSON formatter I used removed spaces in an unintended way. Anything else?

Reply:
Got it back to where we were!